### PR TITLE
Cleanup regional titles

### PR DESCRIPTION
**Pull request by ryanbr**
Remove some unneeded details, also unifi "lists etc" -> "Filterlists".
Also adjust naming for Fanboy's lists.

### CODE REVIEW

File: filter_lists/regional.json [hunk 1]
```diff
@@ -212,7 +212,7 @@
     {
         "uuid": "1C6D8556-3400-4358-B9AD-72689D7B2C46",
         "url": "https://raw.githubusercontent.com/finnish-easylist-addition/finnish-easylist-addition/master/Finland_adb.txt",
-        "title": "Finnish Addition to Easylist",
+        "title": "Easylist Finnish Addition Filterlist",
```
Comment (rmcfadden3) [inline filter_lists/regional.json:215]: @antonok-edm — should this be "Easylist, Finnish Addition to Filterlist" instead? Seems like we need "to" and a comma in there. Or maybe "Finnish Easylist Addition to Filterlist" could work, too.

File: filter_lists/regional.json [hunk 2]
```diff
@@ -527,7 +527,7 @@
     {
         "uuid": "AD3E8454-F376-11E8-8EB2-F2801F1B9FD1",
         "url": "https://raw.githubusercontent.com/tcptomato/ROad-Block/master/road-block-filters-light.txt",
-        "title": "Romanian Ad (ROad) Block List Light",
+        "title": "Romanian Ad (ROad) Fliterlist",
```
Comment (rmcfadden3) [inline filter_lists/regional.json:530]: @antonok-edm — there's a typo here: "Fliterlist" should be "Filterlist"

File: filter_lists/regional.json [hunk 3]
```diff
@@ -542,7 +542,7 @@
     {
         "uuid": "80470EEC-970F-4F2C-BF6B-4810520C72E6",
         "url": "https://easylist-downloads.adblockplus.org/advblock.txt",
-        "title": "RU AdList (Дополнительная региональная подписка)",
+        "title": "RU AdList Fliterlist",
```
Comment (rmcfadden3) [inline filter_lists/regional.json:545]: @antonok-edm — typo here, too

**Review by rmcfadden3 — COMMENTED** [review]
Made comments at lines 215, 530, and 545; the latter 2 address some small typos.

File: filter_lists/regional.json (hunk 3 above)
Comment (ryanbr) [reply filter_lists/regional.json]: resolved, Removed trypo.

File: filter_lists/regional.json (hunk 2 above)
Comment (ryanbr) [reply filter_lists/regional.json]: removed typo

File: filter_lists/regional.json (hunk 1 above)
Comment (ryanbr) [reply filter_lists/regional.json]: Simplified it to 'Easylist Finnish'

**Review by antonok-edm — APPROVED** [review]

**Comment by antonok-edm** [comment]
@ryanbr looks like there are a couple of conflicts

File: filter_lists/regional.json [hunk 4]
```diff
@@ -197,7 +197,7 @@
     {
         "uuid": "1ED1870B-997C-4BFE-AEBC-B67D679BAF3B",
         "url": "https://raw.githubusercontent.com/easylist/easylist/master/fanboy-addon/fanboy_chatapps_third-party.txt",
-        "title": "Fanboy's Anti-chat apps List",
+        "title": "Fanboy's Anti-chat apps",
```
Comment (antonok-edm) [inline filter_lists/regional.json:200]: `Fanboy's Anti-Chat Apps`

File: filter_lists/regional.json [hunk 5]
```diff
@@ -362,7 +362,7 @@
     {
         "uuid": "A0E9F361-A01F-4C0E-A52D-2977A1AD4BFB",
         "url": "https://raw.githubusercontent.com/gioxx/xfiles/master/filtri.txt",
-        "title": "ABP X Files",
+        "title": "ABP X",
```
Comment (antonok-edm) [inline filter_lists/regional.json:365]: - I think `Files` is worth including here
- Also, the "official" title of this one is `NoAds X Files`; not sure if that's a recent change but we should probably follow
- It'd be good to distinguish this as a primarily Italian list too... maybe `NoAds X Files Italian`?

File: filter_lists/regional.json [hunk 6]
```diff
@@ -377,7 +377,7 @@
     {
         "uuid": "03F91310-9244-40FA-BCF6-DA31B832F34D",
         "url": "https://filters.adtidy.org/ios/filters/7.txt",
-        "title": "Adguard Japanese filters (日本用フィルタ)",
+        "title": "Adguard Japanese (日本用フィルタ)",
```
Comment (antonok-edm) [inline filter_lists/regional.json:380]: `AdGuard Japanese 日本用フィルタ`

File: filter_lists/regional.json [hunk 7]
```diff
@@ -137,7 +137,7 @@
     {
         "uuid": "AC023D22-AE88-4060-A978-4FEEEC4221693",
         "url": "https://secure.fanboy.co.nz/fanboy-cookiemonster_ubo.txt",
-        "title": "Easylist-Cookie List - Filter Obtrusive Cookie Notices",
+        "title": "Easylist-Cookie",
```
Comment (antonok-edm) [inline filter_lists/regional.json:140]: `EasyList Cookie`

File: filter_lists/regional.json [hunk 8]
```diff
@@ -17,7 +17,7 @@
     {
         "uuid": "F61D6B7B-4110-4EA4-9C81-38FB4CE90AEC",
         "url": "https://raw.githubusercontent.com/blocklistproject/Lists/master/adguard/porn-ags.txt",
-        "title": "Blocklists Anti-porn list",
+        "title": "Blocklists Anti-porn",
```
Comment (antonok-edm) [inline filter_lists/regional.json:20]: `Blocklists Anti-Porn`

File: filter_lists/regional.json [hunk 9]
```diff
@@ -122,7 +122,7 @@
     {
         "uuid": "0783DBFD-B5E0-4982-9B4A-711BDDB925B7",
         "url": "https://adblock.ee/list.php",
-        "title": "Eesti saitidele kohandatud filter",
+        "title": "Eesti saitidele kohandatud",
```
Comment (antonok-edm) [inline filter_lists/regional.json:125]: `Eesti Saitidele Kohandatud`

File: filter_lists/regional.json [hunk 10]
```diff
@@ -422,7 +422,7 @@
     {
         "uuid": "4E8B1A63-DEBE-4B8B-AD78-3811C632B353",
         "url": "https://raw.githubusercontent.com/EasyList-Lithuania/easylist_lithuania/master/easylistlithuania.txt",
-        "title": "Adblock Plus Lithuania",
+        "title": "Easylist Lithuania",
```
Comment (antonok-edm) [inline filter_lists/regional.json:425]: `EasyList Lithuania`

File: filter_lists/regional.json [hunk 11]
```diff
@@ -572,7 +572,7 @@
     {
         "uuid": "DABC6490-70E5-46DD-8BE2-358FB9A37C85",
         "url": "https://easylist-downloads.adblockplus.org/bitblock.txt",
-        "title": "BitBlock List (Дополнительная подписка фильтров)",
+        "title": "BitBlock",
```
Comment (antonok-edm) [inline filter_lists/regional.json:575]: would be good to call this one out as a Russian list too, i.e. `BitBlock Russian`

File: filter_lists/regional.json [hunk 12]
```diff
@@ -662,7 +662,7 @@
     {
         "uuid": "1BE19EFD-9191-4560-878E-30ECA72B5B3C",
         "url": "https://adguard.com/filter-rules.html?id=13",
-        "title": "Adguard Turkish Filter",
+        "title": "Adguard Turkish",
```
Comment (antonok-edm) [inline filter_lists/regional.json:665]: `AdGuard Turkish`

**Pull request merged**